operator/v1/types_network.go

```diff
@@ -100,12 +100,33 @@ type DefaultNetworkDefinition struct {
 	KuryrConfig *KuryrConfig `json:"kuryrConfig,omitempty"`
 }
 
+// SimpleMacvlanConfig contains configurations for macvlan interface.
+type SimpleMacvlanConfig struct {
+	// master is the host interface to create the macvlan interface from.
+	// If not specified, it will be default route interface
+	// +optional
+	Master string `json:"master,omitempty"`
+
+	// ipam indicates which IPAM module will be used for IP Address Management (IPAM)
+	// now supports 'dhcp'. If it's not specified does it default to dhcp.
+	// +optional
+	IPAM string `json:"ipam,omitempty"`
+
+	// mode is the macvlan mode: bridge, private, vepa, passthru. The default is bridge
+	// +optional
+	Mode MACVLANMode `json:"mode,omitempty"`
+
+	// mtu is the mtu to use for the macvlan interface. Defaults to the value from kernel if unset.
```

Comment:
Which kernel?  Kernel on each node, so different kubelets can be different?

Reply:
Kernel means linux kernel on the node (that runs pod). This "SimpleMacvlan" provides additional network interface that is outside of k8s management. At that time, MTU will be provided from linux kernel and it should be based on parent interface (i.e. master interface) MTU.

Reply:
A better phrasing would probably be "if unset, the host's kernel will select the value."

Reply:
Changed.